build/Dockerfile

```diff
@@ -1,7 +1,19 @@
-FROM alpine:3.8
+#
+# Dockerfile for building local images.
+#
 
-RUN apk upgrade --update --no-cache
+# Build the manager binary
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 as builder
 
-USER nobody
+# Copy in the go src
+WORKDIR /go/src/github.com/kiegroup/kie-cloud-operator
+COPY . .
 
-ADD build/_output/bin/kie-cloud-operator /usr/local/bin/kie-cloud-operator
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o kie-cloud-operator github.com/kiegroup/kie-cloud-operator/cmd/manager
+
+# Copy the controller-manager into a thin image
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+WORKDIR /root/
+COPY --from=builder /go/src/github.com/kiegroup/kie-cloud-operator/kie-cloud-operator .
+ENTRYPOINT ["./kie-cloud-operator"]
```

Comment:
delete this line... the command logic is handled within the operator deployment
`USER nobody` should remain so we aren't running as uid 0